docs/index.md

```diff
@@ -1,5 +1,9 @@
 # Shopsys Framework Knowledge Base
 
+## What is new and how to upgrade
+* For step by step instuctions see [UPGRADE.md](../UPGRADE.md)
+* In [CHANGELOG.md](../CHANGELOG.md) you can find list of all changes in all repositories
```

Comment:
I would say "...you can find the list of all important changes in all repositories maintained in [shopsys/shopsys monorepo](https://github.com/shopsys/shopsys)"